src/controllers/userController.ts

```diff
@@ -139,6 +139,10 @@ async function retrieveUserByEmail(req: Request, res: Response, next: NextFuncti
       attributes: {
         exclude: ['password', 'role'],
       },
+      include: {
```

Comment:
Probably need to include the actual question content too with the respective `UserAnswer` ids